services/power/client/client.go

```diff
@@ -0,0 +1,118 @@
+/* Copyright (c) 2019 Snowflake Inc. All rights reserved.
```

Comment:
Change year to 2023?

Reply:
Thanks, I did forget what year we live in :-)